content/en/docs/reference/scheduling/policies.md

```diff
@@ -17,7 +17,6 @@ You can set a scheduling policy by running
 and using the [Policy type](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1?tab=doc#Policy).
```

Comment:
Could we amend this line as well?

Reply:
Good spot, but that is a config/v1, not config/v1beta1. We have some other config pages coming in.

Reply:
This link is substituted in another PR: #26602